packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -447,6 +459,47 @@ export class TablePageObject<T extends TableRecord> {
         }
     }
 
+    public getSortedColumns(): SortedColumn[] {
+        return this.tableElement.columns
+            .filter(
+                x => !x.sortingDisabled
+                    && typeof x.columnInternals.currentSortIndex === 'number'
+                    && x.columnInternals.currentSortDirection
+                        !== TableColumnSortDirection.none
+            )
+            .sort(
+                (a, b) => a.columnInternals.currentSortIndex!
+                    - b.columnInternals.currentSortIndex!
+            )
+            .map(x => {
+                return {
+                    columnId: x.columnId,
+                    sortDirection: x.columnInternals.currentSortDirection
+                };
+            });
+    }
+
+    public getGroupedColumns(): string[] {
+        return this.tableElement.columns
+            .filter(
+                x => !x.columnInternals.groupingDisabled
+                    && typeof x.columnInternals.groupIndex === 'number'
+            )
+            .sort(
+                (a, b) => a.columnInternals.groupIndex!
+                    - b.columnInternals.groupIndex!
+            )
+            .map(x => x.columnId ?? '');
+    }
+
+    public getChildDisplayCountForGroup(groupRowIndex: number): string {
```

Comment:
I went back and forth here on whether this should return a string or a number. I landed on a string because it was more straightforward to implement, and it provided what was required for the tests I'd like to write. I'm open to updating if reviewers think it would be more appropriate to return a number here.

Reply:
Think I'd lean toward a number since it's a "count".

Reply:
I'd also vote number.